docs/resources/dsm_app.md

```diff
@@ -15,6 +15,34 @@ resource "dsm_app" "app" {
     new_credential = <true/false> 
 ```
 
+## Update the groups
+
+```
+
+variable "app_data" {
+    type = any
+    default = {
+        "mod_group" = {
+            "<group_id>" : "<permissions(separated by '-')>",
+            "<group_id>" : "<permissions(separated by '-')>"
+        }
+    }
+}
+
+
+resource "dsm_app" "app" {
+    name           = <app_name>
+    default_group  = <group_id>
+    other_group    = [<group_id>,<group_id>]
+    description    = <app_description>
+    patch_request  = true
+    mod_group      = "${var.app_data.mod_groups}"
```

Comment:
Let us see if this can be done at create time

Reply:
Made the changes for create also.